util/disk/tempDir_test.go

```diff
@@ -19,42 +19,38 @@ import (
 	"sync"
 	"testing"
 
-	"github.com/pingcap/check"
 	"github.com/pingcap/tidb/config"
+	"github.com/stretchr/testify/require"
 )
 
-func TestT(t *testing.T) {
-	path, _ := os.MkdirTemp("", "tmp-storage-disk-pkg")
+func TestRemoveDir(t *testing.T) {
+	t.Parallel()
+	path, err := os.MkdirTemp("", "tmp-storage-disk-pkg")
+	require.NoError(t, err)
 	config.UpdateGlobal(func(conf *config.Config) {
```

Comment:
It seems a mistake usage about update global config.  You can see this
https://github.com/pingcap/tidb/blob/d7bb71bc1fe21a75cb8eac64493bf7e3407cf806/ddl/serial_test.go#L103
```
	defer config.RestoreFunc()()
	config.UpdateGlobal(func(conf *config.Config) {
		conf.MaxIndexLength = config.DefMaxOfMaxIndexLength
	})
```

And whether we should not make this test case paralleled? Because it will update global config.

Reply:
PTAL @wshwsh12. ^_^

Reply:
/cc @mmyj